apps/prairielearn/src/question-servers/index.js

```diff
@@ -1,19 +1,32 @@
-const _ = require('lodash');
+// @ts-check
```

Comment:
@nwalters512 converting this to TS in a future PR will likely not detect the rename, given the proportion of changes involved. Should I:

* Go straight to TS;
* Create a rename commit and use a merge or rebase commit to master; or
* Create another PR that only converts to TS with minimal changes, then convert this one to build from that?

Reply:
I ended up working on option 3 in #9070. I'll have this PR work directly in TS.